### PR TITLE
Run end-to-end tests daily basis.

### LINKED ISSUE #132: Run `kubewarden-controller` end-to-end tests every day
Add a workflow in the `kubewarden-controller` repository to run end-to-end tests every day to test the latest changes in the repository.

### PR DESCRIPTION
**Pull request by jvanz**
Adds a new workflow to run the Kubewarden end-to-end test every day.

Closes #132

### CODE REVIEW

File: .github/workflows/container-build.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+    branches:
+      - main
+    tags:
+      - 'v*'
+
+
+jobs:
+  build:
+    name: Build
+    uses: jvanz/kubewarden-controller/.github/workflows/container-image.yml@main
```
Comment (jvanz) [inline .github/workflows/container-build.yml:14]: There are some references to my private repo in this PR. There are the action repo and the with the end-to-end tests. This should be changed _before_ merging this PR. I opened it as it is to see if the whole team agree on how we run the tests. Once we are ready, I'll create the repos inside the Kubewarden org, update the repo and merge the code.

Comment (jvanz) [reply .github/workflows/container-build.yml:14]: Once we create the repos, we will be able to version them. Therefore, I will update the reference to the version used. Removing the `main` ref from the files.

File: .github/workflows/e2e-tests.yml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: Kubewarden End-to-end tests
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
```
Comment (jvanz) [inline .github/workflows/e2e-tests.yml:6]: Run every day at midnitght.

File: .github/workflows/e2e-tests.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+name: Kubewarden End-to-end tests
+
+on:
+  workflow_dispatch:
```
Comment (jvanz) [inline .github/workflows/e2e-tests.yml:4]: Allow us to run the workflow from the Github WebUI or via script.

**Review by flavio — APPROVED** [review]
LGTM. I have the impression this looks cleaner than the initial PR you closed :clap:

**Review by viccuad — APPROVED** [review]
LGTM too! thanks!

**Comment by jvanz** [comment]
This is blocked by https://github.com/kubewarden/automation/pull/2

**Comment by flavio** [comment]
> This is blocked by [kubewarden/automation#2](https://github.com/kubewarden/automation/pull/2)

Unblocked

**Pull request merged**